includes/TypesForMembers/MemberEccentricity.js

```diff
@@ -1,20 +1,278 @@
+*/
 function MemberEccentricity(no,
-                            member_start,
-                            member_end,
-                            comment,
-                            params)
-{
-    var member_eccentricity = engine.create_member_eccentricity(no);
-
-    if (typeof member_start !== 'undefined')
+	member_start,
+	member_end,
+	comment,
+	params) {
+    if (arguments.length !== 0) {
+		this.memberEccentricity = createEccentricity(no, member_start, member_end, comment, params);
+		return this.memberEccentricity;
+	}
+}
+
+/**
+* Creates member eccentricity relative to section
+* @param	{Number}	no				Index of member eccentricity, can be undefined
+* @param	{Object}	member_start	Member start
+* @param	{Object} 	member_end		Member end
+* @param	{String}	alignment		Alignment, for more info look at private function setRelativeValues
+* @param	{String}	comment			Comment, can be undefined
+* @param	{Object}	params			Member eccentricity parameters, can be undefined
+* @return	{Object}	Created member eccentricity
+*/
+MemberEccentricity.prototype.RelativeToSection = function (no,
+	member_start,
+	member_end,
+	alignment,
+	comment,
+	params) {
+	this.memberEccentricity = createEccentricity(no, member_start, member_end, comment, params);
+	this.memberEccentricity.specification_type = member_eccentricities.TYPE_RELATIVE;
+	setRelativeValues(this.memberEccentricity, alignment);
+};
+
+/**
+* Creates absolute member eccentricity
+* @param	{Number}	no					Index of member eccentricity, can be undefined
+* @param	{Object}	member_start		Member start
+* @param	{Object} 	member_end			Member end
+* @param	{Number}	ex					Eccentricity in X
+* @param	{Number}	ey					Eccentricity in Y
+* @param	{Number}	ez					Eccentricity in Z
+* @param	{Number}	coordinate_system	Coordinate system, can be undefined. Default value is "Local xyz".
+* @param	{String}	comment				Comment, can be undefined
+* @param	{Object}	params				Member eccentricity parameters, can be undefined
+* @return	{Object}	Created member eccentricity
+*/
+MemberEccentricity.prototype.Absolute = function (no,
+	member_start,
+	member_end,
+	ex,
+	ey,
+	ez,
+	coordinate_system,
+	comment,
+	params) {
+	this.memberEccentricity = createEccentricity(no, member_start, member_end, comment, params);
+	this.memberEccentricity.specification_type = member_eccentricities.TYPE_ABSOLUTE;
+	setAbsoluteValues(this.memberEccentricity, ex, ey, ez, coordinate_system);
+};
+
+/**
+* Creates absolute member eccentricity
+* @param	{Number}	no					Index of member eccentricity, can be undefined
+* @param	{Object}	member_start		Member start
+* @param	{Object} 	member_end			Member end
+* @param	{Number}	ex					Eccentricity in X
+* @param	{Number}	ey					Eccentricity in Y
+* @param	{Number}	ez					Eccentricity in Z
+* @param	{Number}	coordinate_system	Coordinate system, can be undefined. Default value is "Local xyz".
+* @param	{String}	alignment			Alignment, for more info look at private function setRelativeValues
```

Comment:
Katerina is using this function https://github.com/Dlubal-Software/Dlubal_JavaScript_Library/blob/28a5413fc76e93fe72bc61d1ba48323e5bee2893/includes/AnalysisSettings/StaticAnalysisSettings.js#L2 and also some const to switch from human readable text to enumeration.
Please discuss it with her and try to find unified approach